contracts/extensions/RONTransferHelper.sol

```diff
@@ -39,14 +39,14 @@ abstract contract RONTransferHelper {
    * - Consider using `ReentrancyGuard` before calling this function.
    *
    */
-  function _unsafeSendRON(address payable _recipient, uint256 _amount) internal returns (bool _success) {
-    (_success, ) = _recipient.call{ value: _amount }("");
+  function _unsafeSendRON(address payable recipient, uint256 amount) internal returns (bool success) {
+    (success, ) = recipient.call{ value: amount }("");
   }
 
   /**
    * @dev Same purpose with {_unsafeSendRON(address,uin256)} but containing gas limit stipend forwarded in the call.
    */
-  function _unsafeSendRON(address payable _recipient, uint256 _amount, uint256 _gas) internal returns (bool _success) {
-    (_success, ) = _recipient.call{ value: _amount, gas: _gas }("");
+  function _unsafeSendRON(address payable recipient, uint256 amount, uint256 gas) internal returns (bool success) {
```

Comment:
Please consider renaming this method to avoid unintentionally using it instead of the above method

Reply:
renamed.